### PR TITLE
create_database code modification

### PR DESCRIPTION
**Pull request by amoj73**
Added protein bar calories.

### CODE REVIEW

File: create_database.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # Creat food data base
 
-a = {'Chicken':100,'Fish':200,'egg':300}
+a = {'Chicken':100,'Fish':200,'egg':300, 'Protein bar':240}
```
Comment (motelian) [inline create_database.py:3]: Can you add more food items?

**Review by motelian — CHANGES_REQUESTED** [review]
Please add another food item

File: create_database.py (hunk 1 above)
Comment (amoj73) [reply create_database.py]: Added more items.

**Comment by amoj73** [comment]
Added tuna

**Review by motelian — APPROVED** [review]

**Pull request merged**